Tasks/VsTest/constants.ts

```diff
@@ -0,0 +1,16 @@
+export module areaCodes {
+    export const publishResults = 'publishresults';
```

Comment:
const always FULL CAPITALS

Reply:
done

---
In reply to: [136252235](https://github.com/Microsoft/vsts-tasks/pull/5156#discussion_r136252235) [](ancestors = 136252235)